pages/qdm-to-qicore.md

```diff
@@ -1862,27 +1802,17 @@ To capture specific types of "Physical Exam, Recommended" as QICoreObservationNo
 
 ### 8.20 Procedure
 
-QDM defines Intervention and Procedure as:
-* *Intervention* is a course of action intended to achieve a result in
-    the care of persons with health problems that does not involve
-    direct physical contact with a patient. Examples include patient
-    education and therapeutic communication.
-
-* *Procedure* is an act whose immediate and primary outcome
-    (post-condition) is the alteration of the physical condition of the
-    subject. A *procedure* may be a surgery or other type of physical
-    manipulation of a person’s body in whole or in part for purposes of
-    making observations and diagnoses or providing treatment.
+QDM defines Procedure an act whose immediate and primary outcome
```

Comment:
Wording is a little confusing. Suggest changing to "QDM defines Procedure as an act ..."

Reply:
changed as suggested